support-files/mysql.spec.sh

```diff
@@ -430,46 +371,14 @@ Obsoletes:      MySQL-shared-pro-gpl-cert
 Obsoletes:      MySQL-shared-classic MySQL-shared-community MySQL-shared-enterprise
 Obsoletes:      MySQL-shared-advanced-gpl MySQL-shared-enterprise-gpl
```

Comment:
"mysql-wsrep-shared" must have a "Provides: mysql-shared" because that is the RPM capability which applications use to ensure the presence of the shared libraries.